jenkins/deploy.sh

```diff
@@ -103,12 +103,16 @@ $DEPLOY_CMD -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
 SPARK2_PL=${SPARK2_PL:-"spark2-sql-plugin"}
 SPARK2_ART_ID=`mvn help:evaluate -q -pl $SPARK2_PL -Dexpression=project.artifactId -DforceStdout -Dbuildver=24X`
 SPARK2_ART_VER=`mvn help:evaluate -q -pl $SPARK2_PL -Dexpression=project.version -DforceStdout -Dbuildver=24X`
-SPARK2_FPATH="$M2DIR/repository/com/nvidia/$SPARK2_ART_ID/$SPARK2_ART_VER/$SPARK2_ART_ID-$SPARK2_ART_VER"
+SPARK2_FPATH="$M2DIR/com/nvidia/$SPARK2_ART_ID/$SPARK2_ART_VER/$SPARK2_ART_ID-$SPARK2_ART_VER"
 SPARK2_DOC_JARS="-Dsources=${SPARK2_FPATH}-sources.jar -Djavadoc=${SPARK2_FPATH}-javadoc.jar"
 # a bit ugly but just hardcode to spark24 for now since only version supported
 SPARK2_CLASSIFIER='spark24'
-SPARK2_CLASSIFIER_JAR="{$SPARK2_FPATH}-${SPARK2_CLASSIFIER}.jar"
+SPARK2_CLASSIFIER_JAR="${SPARK2_FPATH}-${SPARK2_CLASSIFIER}.jar"
+# Oss requires a main jar file along with classifier jars
```

Comment:
Oss requires a main jar file along with classifier jars

Reply:
this isn't ideal, perhaps we just deploy the jar without classifier then, let me look at more

Reply:
@tgravescs  if so, below changes would work:

```
--- a/jenkins/deploy.sh
+++ b/jenkins/deploy.sh
@@ -103,12 +103,11 @@ $DEPLOY_CMD -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
 SPARK2_PL=${SPARK2_PL:-"spark2-sql-plugin"}
 SPARK2_ART_ID=`mvn help:evaluate -q -pl $SPARK2_PL -Dexpression=project.artifactId -DforceStdout -Dbuildver=24X`
 SPARK2_ART_VER=`mvn help:evaluate -q -pl $SPARK2_PL -Dexpression=project.version -DforceStdout -Dbuildver=24X`
-SPARK2_FPATH="$M2DIR/repository/com/nvidia/$SPARK2_ART_ID/$SPARK2_ART_VER/$SPARK2_ART_ID-$SPARK2_ART_VER"
+SPARK2_FPATH="$M2DIR/com/nvidia/$SPARK2_ART_ID/$SPARK2_ART_VER/$SPARK2_ART_ID-$SPARK2_ART_VER"
 SPARK2_DOC_JARS="-Dsources=${SPARK2_FPATH}-sources.jar -Djavadoc=${SPARK2_FPATH}-javadoc.jar"
 # a bit ugly but just hardcode to spark24 for now since only version supported
 SPARK2_CLASSIFIER='spark24'
-SPARK2_CLASSIFIER_JAR="{$SPARK2_FPATH}-${SPARK2_CLASSIFIER}.jar"
+SPARK2_CLASSIFIER_JAR="${SPARK2_FPATH}-${SPARK2_CLASSIFIER}.jar"
 $DEPLOY_CMD -Durl=$SERVER_URL -DrepositoryId=$SERVER_ID \
             $SPARK2_DOC_JARS \
-            -Dclassifier=$SPARK2_CLASSIFIER \
             -Dfile=$SPARK2_CLASSIFIER_JAR -DpomFile=${SPARK2_PL}/pom.xml

```

Reply:
Maybe we also need to update the nightly build script, as the `SNAPSHOT` jars also contains the classifier `spark24` 
rapids-4-spark-sql-meta_2.11-22.02.0-20220120.205158-1-spark24.jar

Reply:
no I don't want to just update deploy script, we would change the entire module in src. I thought the workaround we did in pom file for classifier worked with deploy as well, but this is using deploy file directly vs mvn deploy or if its just a difference in the repo.

Reply:
ok lets just deploy both and we will leave it documented as using the spark24 jar.